### PR TITLE
9 calculate images overlays

### PR DESCRIPTION
**Pull request by faroukmiled**
I added a function to overlay two images (https://github.com/centralelyon/ntt/blob/9-calculate-images-overlays/src/ntt/frames/overlay_two_images.py)

I added an example and a test (checking if the two input frames have the same shape).

https://github.com/centralelyon/ntt/blob/9-calculate-images-overlays/examples/ex_overlay_images.py
https://github.com/centralelyon/ntt/blob/9-calculate-images-overlays/tests/test_overlay_images.py

Tests passed successfully

### CODE REVIEW

**Comment by faroukmiled** [comment]
@romsson  I made a pull request. Can you check it before committing to main?

**Review by romsson — CHANGES_REQUESTED** [review]

File: examples/ex_overlay_images.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from ntt.frames.overlay_two_images import overlay_images
+if __name__=="__main__":
+    path_frames="./samples"
```
[the remark this answers is not in the document]
Comment (faroukmiled) [reply examples/ex_overlay_images.py]: resolved

File: src/ntt/videos/video_fusion.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from moviepy.editor import VideoFileClip, CompositeVideoClip
+def video_fusion_image(list_videos_path,video_out):
```
[the remark this answers is not in the document]
Comment (faroukmiled) [reply src/ntt/videos/video_fusion.py]: resolved

File: src/ntt/frames/overlay_two_images.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+def overlay_images(path_frames, name_frame1, name_frame2, name_output_frame):
+    path_frame1 = os.join(path_frames, name_frame1)
+    path_frame2 = os.join(path_frames, name_frame2)
+
+    frame1 = cv2.imread(path_frame1)
+    frame2 = cv2.imread(path_frame2)
+
+    overlayed_frame = cv2.addWeighted(frame1, 0.5, frame2, 0.5, 0)
```
[the remark this answers is not in the document]
Comment (faroukmiled) [reply src/ntt/frames/overlay_two_images.py]: resolved

File: src/ntt/videos/video_fusion.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    
+    video_clips = [VideoFileClip(path) for path in list_videos_path]
+    opacities = [0.5 for i in range(len(list_videos_path))]  # Opacity values for each video
```
[the remark this answers is not in the document]
Comment (faroukmiled) [reply src/ntt/videos/video_fusion.py]: resolved

File: examples/ex_video_fusion.py
[the remark this answers is not in the document]
Comment (faroukmiled) [reply examples/ex_video_fusion.py:1]: resolved.

added test_custom function to test_overlay_images.py. The function generates two empty frames and a frame with a green rectangle in the middle (reference frame)
Then we overlay the first empty frame with the frame with a rectangle with 50% opacity for each (we obtain overlayed_image1).
We perform the same operation for the second empty frame  (we obtain overlayed_image2).
We then overlay  overlayed_image1 and overlayed_image2 with 50% opacity for each and we compare the result with reference frame (we should obtain the same result).

**Pull request merged**